### ISSUE TITLE
Coffins Broken

### ISSUE THREAD

**Issue by Sh1tL0rd** [TeamAOF/All-of-Fabric-6#341]
'You can only sleep at night or during thunderstorms'

the whole purpose is to sleep during the day ;-;

**Comment by Linguardium** [comment]
bug in the Dusk config. change min sleep time to 0 in the dusk config

**Comment by Trent2087** [comment]
Yes because then you can sleep in beds at any point in the day as well... What a great option... Sounds like we should yeet this

**Comment by Linguardium** [comment]
actually Dusk's time check doesnt seem to affect vanilla's own time check... it doesnt even listen in the same place or use fapi's allow_sleep_time event

**Comment by Trent2087** [comment]
yeah just tested that to make sure. Odd. Whelp if 0 works, then it works :)
